extensions/amp-base-carousel/1.0/arrow.js

```diff
@@ -85,50 +85,20 @@ export function ArrowNext({customArrow, ...rest}) {
  * @param {!BaseCarouselDef.ArrowProps} props
  * @return {PreactDef.Renderable}
  */
-function DefaultArrow({by, disabled, ...rest}) {
-  // TODO(wg-bento#7): Replace with :hover and :active pseudoselectors.
-  const [hover, setHover] = useState(false);
-  const [active, setActive] = useState(false);
-
-  // Component should reset hover and active states when disabled.
-  useEffect(() => {
-    if (disabled) {
-      setHover(false);
-      setActive(false);
-    }
-  }, [disabled]);
-
+function DefaultArrow({by, ...rest}) {
+  const styles = useStyles();
   return (
     <button
-      style={{
-        ...styles.defaultArrowButton,
-        color: hover ? '#222' : '#fff',
-        transitionDuration: active ? '0ms' : '',
-      }}
+      class={styles.defaultArrowButton}
```

Comment:
We should only use `className` prop for compatibility. A TODO for later: would good to find a linter that would complain about `class` prop.